etpServerExample/MyOwnDataArrayProtocolHandlers.cpp

```diff
@@ -132,11 +132,11 @@ void MyOwnDataArrayProtocolHandlers::on_PutDataArrays(const Energistics::Etp::v1
 
 			auto hdfGroups = tokenize(pdat.second.uid.pathInResource, '/');
 
-			if (hdfGroups.size() != 3) {
+			if (hdfGroups.size() != 4) {
```

Comment:
I don't really understand. The message error below actually shows 3 tokens if we tokenize by slash.
Since this method is buggy, I am not a lot concerned but we should be consistent and with the error message imho.

Reply:
Actually, "/RESQML/groupname/datasetname" is tokenized as follow:
hdfGroups[0]="" + "/" + hdfGroups[1]="RESQML" + "/" + hdfGroups[2]="groupname" + "/" + hdfGroups[3]="datasetname"
Thus, hdfGroups[0] is the empty string preceding the firt delimiter ("/") occurence.

Reply:
Thanks. My mistake.